Assets/Scripts/FlagCapture.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FlagCapture : MonoBehaviour {
+
+    void OnTriggerEnter(Collider hit) {
```

Comment:
not a huge fan of the "hit" variable name - I believe common practice is to use "other"